### PR TITLE
cask for thinlinc

### PR DESCRIPTION
**Pull request by hu1buerger**
Request to include a cask for the Thinlinc Client from Cendio.
This is a vnc over ssh client, with its own server and is actively used.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

**Comment by Hu1buerger** [comment]
@miccal the regex you introduced seems a little loose. Dosnt that increase the risk of an invalid match, therfore causing an invalid version getting extracted?

**Review by hu1buerger — COMMENTED** [review]
zap included

File: Casks/thinlinc-client.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  livecheck do
+    url "https://www.cendio.com/thinlinc/download"
+    regex(/tl[._-]v?(\d+(?:.\d+)+)[._-]client[._-]macos\.iso/i)
+  end
+
+  app "ThinLinc Client.app"
+
+  # zap as discussed https://github.com/Homebrew/homebrew-cask/pull/119225#pullrequestreview-883188247
+  # it appears as thinlinc dosnt persist data anywhere else except saved state
```
Comment (SMillerDev) [inline Casks/thinlinc-client.rb:18]: No need to document this here, the PR will be linked to the commit and is easily searchable in GitHub

**Review by hu1buerger — COMMENTED** [review]
resolved request

@SMillerDev  any changes needed?

File: Casks/thinlinc-client.rb (hunk 1 above)
Comment (Hu1buerger) [reply Casks/thinlinc-client.rb:18]: sure

**Comment by bevanjkay** [comment]
Thank you @Hu1buerger

**Pull request merged**